src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -82,6 +82,10 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
         <service id="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" alias="serializer.normalizer.object" />
+        
+        <service id="serializer.normalizer.flatten_exception" class="Symfony\Component\Serializer\Normalizer\FlattenExceptionNormalizer">
+            <tag name="serializer.normalizer" />
```

Comment:
I would use `-915` as a priority to be consistent with the above specific normalizers.

Reply:
I need to put -800 in order for the normalize method to be executed.